pom.xml

```diff
@@ -39,12 +39,14 @@
         <module>nifi-registry-runtime</module>
         <module>nifi-registry-security-api</module>
         <module>nifi-registry-security-api-impl</module>
-        <module>nifi-registry-framework</module>
+        <module>nifi-registry-security-utils</module>
+	<module>nifi-registry-framework</module>
         <module>nifi-registry-provider-api</module>
         <module>nifi-registry-web-api</module>
         <module>nifi-registry-web-ui</module>
         <module>nifi-registry-bootstrap</module>
         <module>nifi-registry-assembly</module>
+	<module>nifi-registry-client</module>
```

Comment:
Very minor, but if you end up making any other changes to this prior to merge, the indentation is off here and in the dependency added to nifi-registry-web-api's pom.xml.